src/harness/testcases/WINNF_FT_S_EXZ_testcase.py

```diff
@@ -0,0 +1,443 @@
+  loadConfig, generateCpiRsaKeys, generateCpiEcKeys, convertRequestToRequestWithCpiSignature
+
+
+class ExclusionZoneTestcase(sas_testcase.SasTestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def generate_EXZ_1_default_config(self, filename):
+    """Generates the WinnForum configuration for EXZ.1."""
+    # Load device info
+    # Load Devices located outside 50 meters of exclusion zones
+    device_N2_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_N2_1['installationParam']['latitude'] = 40.31304
+    device_N2_1['installationParam']['longitude'] =  -96.25122
+    device_N2_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_N2_2['installationParam']['latitude'] = 40.35491
+    device_N2_2['installationParam']['longitude'] =  -96.4215
+    device_N2_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_N2_3['installationParam']['latitude'] = 40.40931
+    device_N2_3['installationParam']['longitude'] =  -96.09191
+
+    # Load Devices located within at least one exclusion zones
+    device_N3_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_N3_1['installationParam']['latitude'] = 39.6945
+    device_N3_1['installationParam']['longitude'] =  -96.427
+    device_N3_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_N3_2['installationParam']['latitude'] = 39.66068
+    device_N3_2['installationParam']['longitude'] =  -96.72912
+
+    # Load Devices located within 50 meters of at least one exclusion zones
+    device_N4_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_N4_1['installationParam']['latitude'] = 39.96317
+    device_N4_1['installationParam']['longitude'] =  -96.24692
+    device_N4_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_N4_2['installationParam']['latitude'] = 39.96488
+    device_N4_2['installationParam']['longitude'] =  -96.27267
+    device_N4_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_N4_3['installationParam']['latitude'] = 39.96422
+    device_N4_3['installationParam']['longitude'] =  -96.26649
+   
+    #Load Exclusion Zones
+    exz_record_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'exz_record_0.json')))
+    exz_record_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'exz_record_1.json')))
+
+
+    # Forming N2 grant request with overlapping frequency
+    grant_N2_1 = json.load(
+          open(os.path.join('testcases', 'testdata', 'grant_0.json')))   
+    grant_N2_1['operationParam']['operationFrequencyRange'][
+        'lowFrequency'] = 3645000000.0
```

Comment:
Frequencies should be integers, not floating point.

Reply:
fixed